storybook/js/button.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const formButtons = document.querySelectorAll('button[type="submit"]');
```

Comment:
Not an important thing, but you can use just a `buttons` const name because in that context it's not important to differ buttons and formButtons.